aws/sqs/sqs.tf

```diff
@@ -42,3 +42,21 @@ resource "aws_sqs_queue" "deadletter_queue" {
     Terraform             = true
   }
 }
+
+resource "aws_sqs_queue" "reprocess_submission_queue" {
```

Comment:
I'm thinking this should maybe have a redrive policy to the dead letter queue like the reliability queue